README.md

```diff
@@ -1,2 +1,154 @@
-# Admin-CLI
-Client
+Client 1.0.0
+<h1 align="center"> Admin-CLI </h1>
+
+<p align="center">This program is about checking students work in ta's computer and send students score to server by CLI</p>
+
+## Documentation
+* [Installation](#installation)
+* [Getting start](#getting-start)
+  - [Available Commands](#available-commands)
+  - [Login](#login)
+  - [Init work directory](#init-work-directory)
+  - [Start checking work](#start-checking-work)
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install
+
+### library we use that are not in python standard library
+$ pip install requests
+
+$ pip install -U click
+
+Let's start the installation process first you need to install library that we mention 
```

Comment:
suggestion:
To start the installation process. First, you need to install the library that we mentioned, then we will clone this github onto your local computer.